### ISSUE TITLE
Investigate AvD round timing issues

### ISSUE THREAD

**Issue by mikeparker** [fortressforever/fortressforever#60]
hlstriker:the round time on avd maps needs to go
the whole switching thing
and changing the servers timelimit
all avds maps should switch teams, but only when 1 team finishes capping
if the servers timelimit runs out, thats it map changes
it should be up to the server operator to enforce a timelimit

I think there was some similar comments on the forums. Might be worth investigating.

**Comment by squeek502** [comment]
This is from using base_id_new on palermo/cornfield. It was something that Crazycarl made but we never gave it the testing it deserved, so it's kind of an unknown quantity that never got fully realized. Not sure if we should just revert to the old base_id or try to make the new system work.

**Comment by FDA-** [comment]
The weird adjusting of the map time limit is a problem. But I don't think round timers should be removed. Round timers reduce stagnation. I'd much rather switch from D to O after 5 minutes, as opposed to enforcing a shut out for 20. Maybe make the map end after a set amount of rounds, or when the servers static map limit is reached, which ever comes first.

**Comment by alaswell** [comment]
I think that we should revert those maps back to base_id and potentially come back to the round concept if we can nail it down better in the future.

**Comment by squeek502** [comment]
Agreed with both @FDA- and @alaswell. For now, we should probably just revert to base_id, but some sort of straightforward round-timer-system would be nice to have. The current base_id_new is just too confusing at the moment.

**Comment by alaswell** [comment]
So I added a [new branch](https://github.com/fortressforever/fortressforever-scripts/tree/features/id-round-timer) based off of the most recent [objective icon branch](https://github.com/fortressforever/fortressforever-scripts/tree/features/id-defenders-objective) that will add a new round timer system and should fix most, if not all, of this?

**Comment by squeek502** [comment]
@alaswell, I totally missed that comment. Could you create a pull request of that branch and explain the round timer a bit in its description?

**Comment by alaswell** [comment]
Actually, I looked at it just now, and that branch was all jacked up. I think b/c I branched it from the other one, it didn't update correctly or something idk, but it still had those globals in startup() and all sorts of stuff wrong. So, I deleted it. I'll re-make what I had working tonight when my son gives me some time to and create a pull request then.
